apps/src/submit.js

```diff
@@ -0,0 +1,115 @@
+import commonMsg from '@cdo/locale';
+import dom from './dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DialogButtons from './templates/DialogButtons';
+
+var Submit = module.exports = function (options) {
```

Comment:
I kind of wonder if this should behave more like a singleton than a class, as we should never have multiple instances. This would also potentially make it easier to export only the bits we need (afaict, the only methods used externally atm are `setupButtons` and `onSubmitComplete`?